stats/src/main/java/io/airlift/stats/JmxGcMonitor.java

```diff
@@ -132,22 +159,29 @@ private synchronized void onNotification(Notification notification)
             GarbageCollectionNotificationInfo info = new GarbageCollectionNotificationInfo((CompositeData) notification.getUserData());
 
             if (info.isMajorGc()) {
-                majorGcCount.incrementAndGet();
-                majorGcTime.addAndGet(info.getDurationMs());
-                majorGc.add(info.getDurationMs(), MILLISECONDS);
-
-                // assume that major GCs stop the application
-                long applicationRuntime = max(0, info.getStartTime() - lastGcEndTime);
-                lastGcEndTime = info.getEndTime();
+                logMajorGc(info);
+            }
+            else if (info.isMinorGc()) {
+                minorGc.add(info.getDurationMs(), MILLISECONDS);
 
-                log.info(
-                        "Major GC: application %sms, stopped %sms: %s -> %s",
-                        applicationRuntime,
+                // assumption that minor GCs run currently, so we do not print stopped or application time
+                log.debug(
+                        "Minor GC: duration %sms: %s -> %s",
                         info.getDurationMs(),
                         info.getBeforeGcTotal(),
                         info.getAfterGcTotal());
             }
-            else if (info.isMinorGc()) {
+        }
+    }
+
+    private synchronized void onZgcNotification(Notification notification)
+    {
+        if ("com.sun.management.gc.notification".equals(notification.getType())) {
```

Comment:
We can update this to use `com.sun.management.GarbageCollectionNotificationInfo#GARBAGE_COLLECTION_NOTIFICATION`

Also update the existing usage above

Reply:
done